src/sysext/sysext.c

```diff
@@ -262,9 +277,38 @@ static int unmerge_hierarchy(
                 if (r == 0)
                         break;
 
+                r = read_one_line_file(work_dir_info_file, &base64_work_dir);
```

Comment:
you could use read_file_full() with the READ_FULL_FILE_UNBASE64 flag btw

Reply:
Went with cescape/cunescape instead.